CMakeLists.txt

```diff
@@ -51,15 +51,12 @@ endif(NOT DEFINED ENV{KRATOS_INSTALL_PYTHON_USING_LINKS})
 set(CMAKE_INSTALL_RPATH "$ORIGIN/../libs")
 
 # If no test policy enable by default
-if(NOT DEFINED KRATOS_BUILD_TESTING)
-message("-- No KRATOS_BUILD_TESTING is defined. Setting to ON")
-SET (KRATOS_BUILD_TESTING ON)
-endif(NOT DEFINED KRATOS_BUILD_TESTING)
+option(KRATOS_BUILD_TESTING "KRATOS_BUILD_TESTING defines if the C++ tests are compiled. These increases compilation time, but if not set only python tests can be executed. Default setting is ON" ON)
```

Comment:
Can you please add the message back?
I think it is important

Reply:
But I thought the text was informative, do you mean to check the variable and throw the message?

Reply:
Yes

The text as is is only shown in the cmake GUI afaik
It will for sure not be issues